docs/developers/integration.mdx

```diff
@@ -34,9 +37,75 @@ Integrating Connect into your Cosmos SDK application requires a few simple steps
     </Step>
 </Steps>
 
+## Application Configuration
+
+The application node's configuration must be extended, so the oracle configuration can be read into the node through `app.toml`.
+
+The application should contain a custom configuration struct with a `"github.com/cosmos/cosmos-sdk/server/config"` embedded.
+
+Note: application function and type names may vary. The names in the following steps are arbitrary for example purposes only.
+
+```go config.go
+// CustomAppConfig defines the configuration for the app.
+type CustomAppConfig struct {
+	serverconfig.Config
+	// ... other configurations
+	Oracle oracleconfig.AppConfig `mapstructure:"oracle" json:"oracle"`
+}
+```
+
+Next, append the Oracle's default config template to the custom application template.
+
+```go config.go
+func CustomConfigTemplate() string {
+	return serverconfig.DefaultConfigTemplate + oracleconfig.DefaultConfigTemplate
+}
+```
+
+Finally, add a default configuration.
+
+```go config.go
+func DefaultConfig() (string, CustomAppConfig) {
+	serverConfig := serverconfig.DefaultConfig()
+    // edit serverConfig as needed
+
+	oracleCfg := oracleconfig.AppConfig{
+		Enabled:        false,
```

Comment:
Don't we want to default the oracle to enabled?

Reply:
[switch to true](https://github.com/skip-mev/slinky/pull/660/commits/e042ab1c5d08bc6dbab13ee116dc513db28f4ad3)